docs/sample-apps/multimodal-chat/README.md

```diff
@@ -0,0 +1,171 @@
+# Multimodal Chat Application
+
+A full-stack application demonstrating Pixel Table's multimodal processing capabilities. This application allows users to interact with documents, videos, and audio files using natural language queries.
+
+![Screenshot](local_rag_screenshot.png)
+
+## Backend API
+
+### Core Endpoints
+- `GET /` - Service info
+- `GET /health` - Health check
+- `GET /api/files` - List all uploaded files
+
+### File Management
+- `POST /api/upload` - Upload document/video files
+- `POST /api/videos/upload` - Upload video files
+- `POST /api/audio/upload` - Upload audio files
+- `GET /api/videos` - List uploaded videos
+
+### Chat Interface
+- `POST /api/chat` - Send message and get AI response
+
+## Architecture Overview
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant NextJS
+    participant FastAPI
+    participant Pixeltable
+
+    User->>NextJS: Interact with UI (upload files, chat)
+    NextJS->>FastAPI: Make API requests (upload files, get chat response)
+    FastAPI->>Pixeltable: Process files, search documents, generate chat responses
+    Pixeltable-->>FastAPI: Return processed data, search results, chat responses
+    FastAPI-->>NextJS: Respond to API requests
+    NextJS-->>User: Update UI with results
+
+    Note right of Pixeltable: Storing and indexing documents/videos, chunking and embedding text, semantic search, prompting and response generation
+    Note right of FastAPI: File uploads and URL processing, API routing and request handling, integrating with Pixeltable
+    Note right of NextJS: Rendering the user interface, handling user interactions, making API calls to FastAPI, updating the UI with results
+```
+
+## Features
+
+- **Multimodal Data Support**: Process and analyze documents, videos, and audio files
+- **Advanced Processing**: Leverages [Pixeltable's data types and formats](https://docs.pixeltable.com/docs/data-types-and-formats)
+- **Flexible LLM Integration**: 
+  - Default: [OpenAI](https://docs.pixeltable.com/docs/working-with-openai)
+  - Alternative options: [Ollama](https://docs.pixeltable.com/docs/working-with-ollama), [LLama.cpp](https://docs.pixeltable.com/docs/working-with-llamacpp)
+- **Local Processing Options**: Use [Whisper/WhisperX](https://docs.pixeltable.com/docs/whisper) for local transcription
```

Comment:
make it clear that for now it uses OpenAI Whisper but yes can use Whisper/WhisperX instead